dev-requirements.txt

```diff
@@ -3,3 +3,5 @@ sphinx>=1.1
 colour>=0.0.5
 mock>=1.0
 pep8>=1.5.7
+google-api-python-client>=1.4.2
+python-dateutil>=2.4.2
```

Comment:
These two should go into MOCK_MODULES in docs/conf.py: the dev-requirements should  only be used for packages needed to run the tests etc.